### PR TITLE
Card link interactivity

### PR DESCRIPTION
**Pull request by TimMansell**
Generic card:
heading & subtitles underlines on hover
links 1 rem from the bottom.  `card__inner` padding-bottom: 1rem

Staff cards:
Staff name underlines on hover

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 86e12acba7fa1f9aef06bb97faaa79474b4dbbf7

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cdb720f966e7d000895f9bf

File: components/cards/_card-generic.css [hunk 1]
```diff
@@ -69,6 +74,10 @@
       font-weight: var(--fw-semibold);
       text-decoration: underline;
```
Comment (gepifanio) [inline components/cards/_card-generic.css]: I think we can remove the underline prop of the `sub-titles` of the cards

File: components/cards/_card-generic.css [hunk 2]
```diff
@@ -69,6 +74,10 @@
...
       color: rgb(var(--col-background-primary-navy));
+
+      &:hover {
```
Comment (gepifanio) [inline components/cards/_card-generic.css]: And will not be necessary to use this :)

File: components/cards/_card-generic.css (hunk 1 above)
Comment (TimMansell) [reply components/cards/_card-generic.css]: It's strange, I can't see this line when checking out this branch on my local.  Can checkout this branch and see if you can see it?

File: components/cards/_card-generic.css (hunk 2 above)
Comment (TimMansell) [reply components/cards/_card-generic.css]: I have removed text underline for the `<a>` links.  We'll still need the `:hover` to remove it thought.

Comment (gepifanio) [reply components/cards/_card-generic.css]: Hmm, ok 👍

**Comment by jaredh-um** [comment]
Looks good. Might need to look at those link colours for the UoM version but we can do that down the track.

**Pull request merged**